### DISCUSSION TITLE
无效：osc_lazy.lua里的timetotal=false比较酷

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#17] [回收]
一般点开电影的时候就看到总时间了(明白看完大概要多久),而看的过程中如果去碰进度条,一般是想知道还剩下多久,所以我一直觉得之前那个剩余时间很好,一目了然,然而osc_lazy.lua里默认值被改了...

**Comment by hooke007** [comment]
我并没有阻止脚本的选项设定，你可以在对应的配置文件中自行修改这些可定义选项。
之所以修改默认值是因为....我自己不喜欢看剩余时间

**Reply by yeezylife** [reply]
_(如果可能的话...改变习惯的变化最好少一点,因为我这种笨笨有时候自己调不回去..)_

**Reply by hooke007** [reply]
>改变习惯的变化

对我来讲，mpv的默认设置是“反人类的和不合习惯的”，因为从迁移到mpv之前我用的播放器默认都是显示总时间

>自己调不回去

正常提问即可

**Reply by yeezylife** [reply]
> 正常提问即可

OK
